gammapy/background/make.py

```diff
@@ -46,6 +51,11 @@ def make_bg_cube_model(observation_table, fits_path, method='default'):
         Path to the data files.
     method : {'default', 'michi'}, optional
         Bg cube model calculation method to apply.
+    do_not_force_mev_units : bool, optional
+        Set to `True` to use the same energy units as the energy
+        binning for the bg rate. (Only applicable to the *default*
+        method; the *michi* always enforces `MeV` in the bg rate
+        units.)
```

Comment:
Can you make this an option `unit='1 / (MeV sr s)'` that is applied for any method?
Simpler, more consistent (across methods) and more flexible, no?